scripts/e2e.oasis.sh

```diff
@@ -145,10 +159,11 @@ start_web3
 
 printf "${GRN}### Depositing tokens to runtime...${OFF}\n"
 
-deposit 1000
+deposit 1000000000000
 
-printf "${GRN}### Running web3 tests implementation...${OFF}\n"
+# printf "${GRN}### Running web3 tests implementation...${OFF}\n"
 
-run_tests
+#run_tests
```

Comment:
I've been using this only to setup the local network and the paratime, and was then running my own tests against it, so this part should be reverted.

Reply:
Yeah, this should be removed and IMO CI should be made working, even if it's just some basic tests.